server/middlewares/index.js

```diff
@@ -0,0 +1,47 @@
+import jwt from 'jsonwebtoken';
+import db from '../models';
+
+const { User } = db;
+/**
+ *Users endpoint middlewares
+ *
+ * @export
+ * @class UsersMiddleware
+ */
+export default class UsersMiddleware {
+  /**
+   *Validates token
+   * @returns {undefined}
+   * @static
+   * @param {*} req
+   * @param {*} res
+   * @param {*} next
+   * @memberof UsersMiddleware
+   */
+  static async verifyToken(req, res, next) {
+    const token = req.headers.authorization;
+    if (!token) {
+      return res.status(400).send({
+        message: 'Token is not provided'
+      });
+    }
+    try {
+      const decoded = await jwt.verify(token, process.env.SECRET);
```

Comment:
You should leverage the existing helper functions that  handle token verification (`verifyToken`) and returning responses (`successResponse`, `errorResponse`)